src/config/keys.ts

```diff
@@ -282,6 +282,17 @@ export const buildActionsKey = (args: {
   },
 ];
 
+export const buildMemberActionsFilteredByDateKey = (args: {
```

Comment:
```suggestion
export const buildMemberActionsKey = (args: {
```
I think on longer term we will add more filters, so no need to be too precise.